README.md

```diff
@@ -35,667 +35,45 @@ Currently k8s-cleaner has rich set of working examples to identify and list unus
-                elseif kind == "HorizontalPodAutoscaler" then
-                    table.insert(autoscalers, resource)
-                end
-        end
-
-        -- Check for each deployment if there is a matching HorizontalPodAutoscaler
-        for _, deployment in ipairs(deployments) do
-            local deploymentName = deployment.metadata.name
-            local matchingAutoscaler = false
-
-            for _, autoscaler in ipairs(autoscalers) do
-                if autoscaler.spec.scaleTargetRef.name == deployment.metadata.name then
-                    matchingAutoscaler = true
-                    break
-                end
-            end
-
-            if not matchingAutoscaler then
-                table.insert(deploymentWithNoAutoscaler, {resource = deployment})
-                break
-            end
-        end
-
-        if #deploymentWithNoAutoscaler > 0 then
-          hs.resources = deploymentWithNoAutoscaler
-        end
-        return hs
-      end
-```
-
-## Delete Kubernetes resources on a configured time to live or expiration date
-
-Finds resources that have the __cleaner/ttl__ annotation, which specifies their maximum lifespan. Deletes resources that have lived longer than their specified TTL.
-YAML can be found [here](https://github.com/gianlucam76/k8s-cleaner/blob/main/examples-unused-resources/time_based_delete/delete_resource_based_on_ttl_annotation.yaml).
-
-Find resources that have the __cleaner_expires__ annotation, which specifies their expiration date. Deletes resources that have that have surpassed their expiration date.
-YAML can be found [here](https://github.com/gianlucam76/k8s-cleaner/blob/main/examples-unused-resources/time_based_delete/delete_resource_based_on_expire_date.yaml).
-
-## DryRun 
-
-To preview which resources match the __Cleaner__'s criteria, set the __Action__ field to _Scan_. The Cleaner will still execute its logic but will not actually delete or update any resources. To identify matching resources, search the controller logs for the message "resource is a match for cleaner".
-
-```yaml
-apiVersion: apps.projectsveltos.io/v1alpha1
-kind: Cleaner
-metadata:
-  name: cleaner-sample1
-spec:
-  schedule: "* 0 * * *" # Runs every day at midnight
-  resourcePolicySet:
-    resourceSelectors:
-    - namespace: test
-      kind: Deployment
-      group: "apps"
-      version: v1
-      labelFilters:
-      - key: serving
-        operation: Equal
-        value: api # Match deployments with the "serving" label set to "api"
-      - key: environment
-        operation: Different
-        value: prouction # Match deployments with the "environment" label different from "production"
-  action: Scan
-```
-
-By setting __Action__ to _Scan_, you can safely test the Cleaner's filtering logic without affecting your actual deployment configurations. Once you're confident in the filtering criteria, you can set _Action_ to delete or modify.
-
-## Schedule
```

Comment:
We need to add a section about those fields in documentation. I feel few will look at CRD field comment to figure out how to set it.